sdk/build.gradle

```diff
@@ -67,7 +67,7 @@ dependencies {
     implementation "androidx.appcompat:appcompat:1.4.0"
     implementation "androidx.core:core-ktx:1.7.0"
     implementation "com.google.android.gms:play-services-ads-identifier:17.1.0"
-    implementation "com.google.android.gms:play-services-location:18.0.0"
+    implementation "com.google.android.gms:play-services-location:21.0.0"
```

Comment:
Just calling out that the [release notes](https://developers.google.com/android/guides/releases#october_13_2022) say to avoid `21.0.0`, but it probably doesn't affect us since we don't use LocationSettingsRequest.  Though maybe other uses outside of Radar could be affected.

Reply:
I backed it down to 18.0.0. Code change can remain, future-compatible, but we can wait to bump the dependency version. Mind reviewing again?